scipy/stats/tests/test_stats.py

```diff
@@ -5792,6 +5799,93 @@ def test_weights_masked_1d_array(self):
         check_equal_gmean(a, desired, weights=weights, rtol=1e-5)
 
 
+class TestPowMean:
```

Comment:
Where are the values coming from?

Reply:
Currently, from my computer, because I have not found numerical examples for power mean, and weighted power mean.

Reply:
mmm ok so you ran the function and noted these values? We should try to either find some reference cases or make simple ones which we would know the values.

Reply:
Great update, could you make some parametrization maybe? It would be easier to follow.

Reply:
>Please let me resolve my comments

I’m really sorry about that!

Ok, I have updated tests, parametrizing tests which share similar elements.